recipe/meta.yaml

```diff
@@ -110,7 +110,7 @@ outputs:
 
   - name: cuda-cupti-static
     build:
-      skip: True  # [not linux]
+      skip: True  # [not linux64]
```

Comment:
It looks like the tests for `cuda-cupti-static` fail on `linux_aarch64` & `linux_ppc64le` as the static library is missing. Looking at the `redist`s for both of these, it appears there isn't a static library shipped. So just disabled `cuda-cupti-static` on `linux_aarch64` & `linux_ppc64le`.